bin/agd

```diff
@@ -29,18 +29,18 @@ done
 function diffsha1() {
```

Comment:
Please save future readers some time as the functionality and its corner case behavior are unusual.

```suggestion
# diffsha1 stamp_name find_args...
# executes find(1) with the given arguments and saves the resulting file list,
# then runs $DIGEST_CMD on the file list, saves the digest to file stamp_name.new,
# and compares to the saved digest in the file stamp_name.
# Returns success (0) if it cannot write to the temporary files, if the find results are empty,
# or if the computed digest is the same as the saved digest.
# Returns failure (1) if the computed digest is different.
function diffsha1() {
```